3rdparty/snmalloc/allocator.cpp

```diff
@@ -22,8 +22,7 @@ void* ThreadAllocUntyped::get()
 #define OPEN_ENCLAVE
 #define SNMALLOC_SGX
 #define USE_RESERVE_MULTIPLE 1
```

Comment:
```suggestion
```

Reply:
This line is not required anymore.

Reply:
Yes, that's absolutely my mistake, this was right in https://github.com/achamayou/openenclave/pull/1 but it was changed accidentally and incorrectly when I did this PR.